### ISSUE TITLE
[Request] Add a search / find in Highlighted Messages Section

### ISSUE THREAD

**Issue by ripslayer** [chatty/chatty#243]
As the title says if you are following lot's of channels through chatty and if you have lot's oh highlighted words and you check those Highlighted Message section after few hrs you get to see lot's of highlighted messages now it would be useful if you can add a find / search so we can search/find some specific words from the Highlighted Message section itself. 

Thank you

**Comment by OrangutanGaming** [comment]
So like some kind of table where you can see all the highlighted words with filters and a search?

**Comment by ripslayer** [comment]
Yes, add kinda filter or similar to how find works on a webpage or documents. So yeah
